lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm

```diff
@@ -133,16 +133,22 @@ sub get_levels {
 
 sub determine_genome_build {
   my $self = shift;
-  my $ref_build_name1 = shift;
-  my $genome_build = "";
-  my $b37_build =
-    Genome::Model::Build::ReferenceSequence->get(name => "GRCh37-lite-build37");
-  my $ref_build1 = Genome::Model::Build::ReferenceSequence->get(name => $ref_build_name1);
-  if($b37_build->is_compatible_with($ref_build1)) {
-    return "b37";
+  my $supplied_build_name = shift;
+
+  my $canonical_b37_build = Genome::Model::Build::ReferenceSequence->get(name => "GRCh37-lite-build37");
+  my $canonical_b38_build = Genome::Model::Build::ReferenceSequence->get(name => "GRC-human-build38");
+  my $supplied_build = Genome::Model::Build::ReferenceSequence->get(name => $supplied_build_name);
+  my $igv_named_build = '';
+  if($canonical_b37_build->is_compatible_with($supplied_build)) {
+    $igv_named_build = "b37";
+  } elsif ($canonical_b38_build->is_compatible_with($supplied_build)){
+    $igv_named_build = "hg38";
+  }
```

Comment:
Along similar lines as @susannasiebert's suggestion - something like this could make adding new references easier in the future. 

```perl
my %reference_to_igv_map = {
    'GRCh37-lite-build37' => 'b37'
    'GRC-human-build38' => 'hg38'
};

my $supplied_build = Genome::Model::Build::ReferenceSequence->get(name => $supplied_build_name);
my $igv_named_build;

while (($reference_name, $igv_name) = each %reference_to_igv_map) {
    if (my $reference_build = Genome::Model::Build::ReferenceSequence->get(name => $reference_name) &&
        $reference_build->is_compatible_with($supplied_build)) {
            $igv_named_build = $igv_name;
            last;
    }
}
````

Otherwise looks good to me!